test/functional/bundleadd_v2/add-client-certificate.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_dir_path=$(dirname "$(realpath "$TEST_NAME")/$TEST_NAME")
```

Comment:
This command causes the current failures in Travis. I am removing this and storing the keys in /tmp.